repotracker/repotracker.go

```diff
@@ -399,7 +399,8 @@ func (repoTracker *RepoTracker) GetProjectConfig(ctx context.Context, revision s
 		_, ymlFmtErr := errors.Cause(err).(thirdparty.YAMLFormatError)
 		_, noFileErr := errors.Cause(err).(thirdparty.FileNotFoundError)
 		parsingErr := strings.Contains(err.Error(), "translating project")
-		if apiReqErr || noFileErr || ymlFmtErr || parsingErr {
+		mergingErr := strings.Contains(err.Error(), "merging project configs")
```

Comment:
Although I'm the one who did this in the first place, we should avoid hardcoding errors when we can. Instead it makes sense to make a constant for this error and for translating (like [this one](https://github.com/evergreen-ci/evergreen/blob/04bd7460eadc19cb52b146e5ccbfe3d4edbc39bc/model/project_parser.go#L25) and use it to wrap the original error and to check the error here.

Reply:
done